src/pages/uxp/changelog3P.md

```diff
@@ -0,0 +1,25 @@
+<!--
+index_desc: Change Log in UXP version
+-->
+
+# UXP Changelog
+
+## UXP v6.3
```

Comment:
I'm not sure if this is actually for this UXP version, but I think the addition of the `FileSystem` APIs needs to be in this changelog (as from a 3P perspective, this is in "the new release").

Reply:
🟡

Reply:
Tracked in #72